src/main/java/com/liferay/faces/util/render/internal/FacesURLEncoderMojarraImpl.java

```diff
@@ -35,11 +35,11 @@
 	// Logger
 	private static final Logger logger = LoggerFactory.getLogger(FacesURLEncoderMojarraImpl.class);
 
-	// Private Constants
-	private final Method MOJARRA_METHOD_WRITE_URL;
+	// Private Final Data Members
+	private final Method mojarraMethodWriteURL;
```

Comment:
@stiemannkj1 As I recall, usage of the final keyword for private data members is not commonly used in the codebase.